ubuntu/common/cleanup_insecure_packages.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Remove packages requiring Ubuntu Pro for security updates
+apt-get update
```

Comment:
suggest a more general name since this might be used for other purposes later. `cleanup_unused_packages.sh` or `remove_unused_packages.sh`

Also, I'm not sure we need to do another update here. It happens in `install_prerequisites.sh` a few lines before. I suppose it doesn't hurt, but seems redundant.